examples/cryptocurrency/src/lib.rs

```diff
@@ -283,7 +283,8 @@ pub mod contracts {
 /// Cryptocurrency API implementation.
 pub mod api {
     use exonum::crypto::PublicKey;
-    use exonum_rust_runtime::api::{self, ServiceApiBuilder, ServiceApiState};
+    use exonum_api as api;
```

Comment:
This direct dependency could be avoided if `Error` had `not_found` constructor and if `Result` as continued to be re-exported.